index.html

```diff
@@ -19,6 +19,7 @@
         <span class="title">To Do</span>
       </header>
       <div id="todoWrapper">
+        <!-- asidebar -->
```

Comment:
this comment is unneccesary. Please remove it before merge

Reply:
will do it @ashishsingl right away!